### ISSUE TITLE
how to disable // swiftlint:disable

### ISSUE THREAD

**Issue by hishamdubizzle** [realm/SwiftLint#3129]
Hi,
    We have code base passings swiftlint checks. but I found that code passed checks not because code is clean. It is because some developers disabled checks one by one in code files using // swiftlint:disable

M question or suggestion, how to run swiftlint ignoring any // swiftlint:disable used in code files ?

Thank you

**Comment by jpsim** [comment]
You could write a custom rule that looks for disable commands. However I’d encourage a less extreme approach, like perhaps code review or talking to your teammates about when it’s appropriate to disable lint checks.

**Comment by hishamdubizzle** [comment]
at least SwiftLint report should show errors that are commented. or swiftlint should support a command line argument that ignores "disable" to show real swiftlint report with all errors

**Comment by the-real-adammork** [comment]
you could remove all those comments then run `swiftlint` 

```console
$ grep -r */** --include '*.swift' -le "swiftlint:disable all" |  xargs -I % sed -i -e 's;^\/\/  swiftlint\:disable all;;' %

$ swiftlint
```

**Comment by hishamdubizzle** [comment]
Thanks for response
